internal/generator/vector/output/splunk/splunk.go

```diff
@@ -95,11 +95,11 @@ func sink(id string, o obs.OutputSpec, inputs []string, index string, secrets ob
 
 func FixTimestampFormat(componentID string, inputs []string) Element {
 	var vrl = `
-ts, err = parse_timestamp(.@timestamp,"%+")
+.timestamp, err = parse_timestamp(.@timestamp,"%+")
```

Comment:
If I understand correctly, then this in the sink. Shouldn't all entries have both `.timestamp` and `.@timestamp` at this point already?

Reply:
`timestamp` comes from vector, and depending on source/sink is not always the value or format we want.   I assumed the parse was necessary here, so setting the new `timestamp` field to the same we already have VRL for (`@timestamp`) .    It's possible this formatting is not necessary for splunk.   I can try it again and see what splunk is expecting and/or accepts.

Reply:
Seeing as VRL is typed, I would expect `.timestamp` to have a type of "timestamp" (think `time.Time` in Go) as opposed to being an opaque type ("string"), meaning that it should possible for the sink to convert it to whatever representation it needs, be that, for example, a unix-timestamp or something RFC3339 formatted.